### ISSUE TITLE
Enabling HA causes error and reboot of hosts

### ISSUE THREAD

**Issue by john-arvid** [xcp-ng/xcp#143]
Having a problem when enabling HA.

xcp-ng version: 7.4
xcp-ng center 7.6.3

I have two hosts in a cluster, they use a shared nfs storage. (I just changed the ip of the SR, removed and created new pbd's)

When I try to configure HA it fails with: Enabling HA on pool 'HP Cluster'Internal error: (Failure "Unrecognised MTC exit code: 134")
Then it takes 2 minutes and both hosts do a force reboot.

I tried to check the logs, but the log partition was full. (Might be related or another problem)

But I got a crash log on the pool master. But I am not sure where to look for information.
I have extracted it with the Server Status Report tool.

**Comment by olivierlambert** [comment]
Hi!

Please use a more recent version of XCP-ng, 7.4 is not maintained anymore.

**Comment by john-arvid** [comment]
Hi, thanks for your reply.

1. I was wrong, I am using 7.4.1. Not sure if that one is maintaned anymore? (I am working on upgrading to 7.6)
2. Ran ntpq -p and got several remotes
3. Wish I had 3 hosts, but for now that needs to be a wish

**Comment by stormi** [comment]
Only the version before the latest is maintained at the moment, so that means 7.5 and 7.6.

**Comment by john-arvid** [comment]
Ok, will upgrade and report back.

**Comment by john-arvid** [comment]
Upgraded to 7.6 using cli and now the enabling of HA does not cause any error.

**Comment by olivierlambert** [comment]
Thanks for your feedback!

**Comment by john-arvid** [comment]
This problem happended again, and this time I everything should be updated.

I had to disable HA to do some maintenance on one server, some days after it came back I tried to enable HA.
Same error as before and all the servers to a force reboot.

I have the crash logs but I am not sure where to look.

**Comment by john-arvid** [comment]
@olivierlambert Should I create a new ticket or leave this open?

**Comment by olivierlambert** [comment]
Please double check you have applied all yum update, then rebooted. Also triple check you have NTP correctly running and all hosts at the same time.

A strange behavior on 2 hosts is sadly not very surprising but still…

**Comment by john-arvid** [comment]
Ok, I will do that. Just have to clean /var/log, was to full to do any updates. (updates were last done the 15th of february)

What do you mean? Is this expected behaviour? And I don't see what this has to do with 2 hosts, if I had 5 hosts all of them would have been restarted.

**Comment by olivierlambert** [comment]
No I mean HA support on less than 3 hosts is not officially supported by Citrix.

**Comment by john-arvid** [comment]
I have now cleaned the log folder and updated with yum.
Also did a restart and confirmed that NTP is ok.

I can enable HA without any problems now. 
I have also tried to disable it, enter and exit maintenance mode, then to enable HA again. All without any problems.
The only thing I haven't done is to fill /var/log to 100% as it was before.
Maybe this was the reason for the error?

Is it normal that /var/log should use 100% ?
